tests/tw-merge.benchmark.ts

```diff
@@ -1,25 +1,129 @@
-import { bench, describe } from 'vitest'
+import { afterAll, bench, describe } from 'vitest'
 
 import { extendTailwindMerge } from '../src'
 
 import testDataCollection from './tw-merge-benchmark-data.json'
 
 type TestDataItem = Exclude<(typeof testDataCollection)[number][number], true>[]
 
+interface MemoryStats {
+    heapUsed: number
+    heapTotal: number
+    external: number
+    rss: number
+}
+
+function getMemoryUsage(): MemoryStats {
+    const usage = process.memoryUsage()
+    return {
+        heapUsed: usage.heapUsed,
+        heapTotal: usage.heapTotal,
+        external: usage.external,
+        rss: usage.rss,
+    }
+}
+
+function formatBytes(bytes: number): string {
+    if (bytes === 0 || !Number.isFinite(bytes)) return '0 B'
+    const k = 1024
+    const sizes = ['B', 'KB', 'MB', 'GB']
+    const i = Math.floor(Math.log(Math.abs(bytes)) / Math.log(k))
+    return `${(bytes / Math.pow(k, i)).toFixed(2)} ${sizes[i]}`
+}
+
+/**
+ * Forces garbage collection if available (requires Node.js --expose-gc flag).
+ * Used to establish clean memory baselines for accurate memory measurements.
+ */
+async function forceGarbageCollection(): Promise<void> {
+    if (typeof globalThis.gc === 'function') {
+        await globalThis.gc()
+    } else {
+        // eslint-disable-next-line no-console -- This is a warning that will be printed to the console if garbage collection is not exposed.
+        console.warn(
+            'Garbage collection not exposed. Run with --expose-gc for accurate memory measurements.',
+        )
+    }
+}
+
+// Guard to ensure memory summary is printed exactly once. teardown() runs after every
+// benchmark iteration, so without this guard the summary would be printed multiple times
+// (once per iteration of the last benchmark).
+let summaryPrinted = false
```

Comment:
Looks like this variable is not used anywhere. Maybe forgot to add the logic or forgot to remove?